examples/rust_debug/src/build.rs

```diff
@@ -0,0 +1,6 @@
+fn main() {
+    // link to ld.lib.so in debug build mode (needed for wait_for_continue).
+    #[cfg(debug_assertions)] {
+        println!("cargo:rustc-link-arg=-l:ld.lib.so");
```

Comment:
> While the idea is good in principle, the build script is not necessary, because ld.lib.so is already included as a symbol of the base dependency.

Oh, that's good to know! Will help me simplify a few things on my end as well : ) Thanks!